Trainer_finetune.py

```diff
@@ -30,18 +36,28 @@ def device(self):
         self.net.to(torch.device("cuda"))
 
     def load_model(self, name=None, rank=0, real=False):
-        def convert(param):
-            return {
-            k.replace("module.", ""): v
-                for k, v in param.items()
-                if "module." in k and 'attn_mask' not in k and 'HW' not in k
-            }
         if rank <= 0 :
             if name is None:
                 name = self.name
             print(f"loading {name} ckpt")
             self.net.load_state_dict(convert(torch.load(f'ckpt/{name}.pkl')), strict=True)
 
+    def from_pretrained(self, model_name):
+        try:
+            from huggingface_hub import hf_hub_download
+
+            ckpt_path = hf_hub_download(
+                repo_id="MCG-NJU/VFIMamba", filename="ckpt/" + model_name + ".pkl"

```

Comment:
Note, it would still be useful to store the checkpoints in separate model repos (one for VFIMamba and one for VFIMamba_S), this to ensure downloads work: https://huggingface.co/docs/hub/models-download-stats (if you're interested in seeing how many times people use your model)

Reply:
Thank you for your advice! Pushed a new version.